app/models.py

```diff
@@ -11,3 +11,11 @@ class ChatResponse(BaseModel):
 
 class LearnRequest(BaseModel):
     text: str = Field(description="The text to be learned by the chatbot")
+
+#Practice
+class BMIRequest(BaseModel):
+    weight: float = Field(description="Weight value requested to calculate BMI.")
+    height: float = Field(description="Height value requested to calculate BMI.")   
+    
+class BMIResponse(BaseModel):
+    message: str = Field(description="BMI value to be responded by the chatbot")
```

Comment:
You don't have to update this now, but in general, backend engineers try to design APIs that are as generic as possible so that different kinds of clients (browsers, other servers) use them.

In this case, I would just return a float number and ask clients, such as browsers, to be responsible for formatting on UI, appending messages, or showing in colors.